example/hyprland.conf

```diff
@@ -190,17 +184,17 @@ bindm = $mainMod, mouse:272, movewindow
 bindm = $mainMod, mouse:273, resizewindow
 
 # Media Control keys
-#! Dependencies: pulseaudio-ctl brightnessctl playerctl
+# Dependencies: pulseaudio-ctl brightnessctl playerctl
```

Comment:
Maybe also specify those are *optional* dependencies, for if people desire to use it.
Even better, I would remove this section entirely and put it under a tips and tricks page on the wiki.